docs/usage.md

```diff
@@ -17,7 +17,7 @@ TopoStats will scan for all images within this directory but currently it will o
 (i.e. `.spm` _or_ `.jpk` _or_ `.asd`). This may change in the future.
 
 
-## Running TopoStats
+## Running TopoStats - Test Run
```

Comment:
Why is this a test run?

Reply:
Because it'll run the default file which is configured to use tests. This is before you've explained how to modify the file so thought it'd make more sense to guide people and understand whats happening.